content/millennium/r4/conformance/structure-definition.md

```diff
@@ -151,6 +152,7 @@ Authorization is not required.
 [`performing-location`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location?_format=json
 [`period`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/period?_format=json
 [`pharmacy-verification-status`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/pharmacy-verification-status
```

Comment:
Do we want to update this URL to include `?_format=json`?

Reply:
Nah, it's not something we've ever touched so let's leave it.